src/EnergyPlus/SZVAVModel.cc

```diff
@@ -846,7 +850,7 @@ namespace SZVAVModel {
                                                                                           AirMassFlow,
                                                                                           0.0,
                                                                                           maxAirMassFlow,
-                                                                                          CoolingLoad,
+                                                                                          CoolingLoad, // passing a bool but expecting a float
```

Comment:
Type mismatch between `CoolingLoad` which is a bool, and the argument type, which is a float.